### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by Jhaman1895**
## In this Milestone, I have taken the following steps to create the desktop version of template-2 from the given Figma design:
- [  ] Used the mobile-first responsive design principle.
-  [  ] Implemented a media query and adjusted the markup accordingly to transition from the mobile to the desktop version.
-  [  ]  Implemented CSS grid to layout the recent works section and CSS flexbox to layout the rest.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +76,25 @@ <h4>Multi-Post Stories</h4>
...
+            unknown printer took a standard dummy text.
+            </p>
+            <ul class="tech-stack">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+            </ul>
+            <button type="button">See Project</button>
+          </div>
```
Comment (KayLemba) [inline index.html:96]: While you did a nice job working on the project requirements. There are still some issues you need to correct. The content of this part of the page seems to be overlaping. Please correct this and carefully inspect the figma template. 

This is your design:
Screenshot from 2022-07-16 20-38-59

Follow this design:
Screenshot from 2022-07-16 20-40-47

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Jhaman1895   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Jhaman1895** [comment]
## In this Milestone, I have taken the following steps to create the desktop version of template-2 from the given Figma design:
- [  ] Used the mobile-first responsive design principle.
-  [  ] Implemented a media query and adjusted the markup accordingly to transition from the mobile to the desktop version.
-  [  ]  Implemented CSS grid to layout the recent works section and CSS flexbox to layout the rest.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Jhaman1895        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- No linter errors 
- Neat and Nice User interface 


Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**